lib/pkgcloud/openstack/client.js

```diff
@@ -195,6 +195,10 @@ Client.prototype._request = function (options, callback) {
       self.emit('log::trace', 'Creating Authenticated Proxy Request');
       var apiStream = Client.super_.prototype._request.call(self, options, callback);
 
+      proxyStream.on('abort', function () {
+        apiStream.abort();
+      });
+
```

Comment:
@meteormatt here you could add the `abort()` function which will be on the returned `proxyStream` from `this._request` in `exports.download`.

```js
proxyStream.abort = function () {
  apiStream.abort();
});
```